lib/content-services/src/lib/dialogs/dialog/dialog.component.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
```

Comment:
old copyright, should be Copyright © 2005-2024

Reply:
yes, it will be updated with rebase